### PR TITLE
Added shortcut support to open TabWrangler popup

### PR DESCRIPTION
**Pull request by xiio**
This PR introduces a keyboard shortcut feature that allows users to quickly open the TabWrangler popup without clicking the toolbar button.

### CODE REVIEW

**Comment by ssorallen** [comment]
Thank you for the pull request. Chrome already adds a shortcut for all extensions that opens the pop up (called "Activate the extension" in keyboard shortcuts). Have you tried that, or are you a Firefox user?

Screenshot 2024-12-23 at 9 35 42 AM

**Comment by dr-tomasz** [comment]
Hello @ssorallen! 

Thank you for your reaction! I noticed that during test, but I'm Firefox user and unfortunately 🦊 is missing this feature by default…  🤷

**Comment by xiio** [comment]
Hello @ssorallen!

Thank you for your reaction! I noticed that during test, but I'm Firefox user and unfortunately 🦊 is missing this feature by default… 🤷

File: app/manifest.template.json [hunk 1]
```diff
@@ -21,6 +21,9 @@
     },
     "wrangle-current-tab": {
       "description": "__MSG_commandWrangleCurrentTab__"
+    },
+    "_execute_action": {
+      "description": "__MSG_commandWrangleOpenPopup__"
```
Comment (ssorallen) [inline app/manifest.template.json:26]: There are Chrome- and [Firefox-specific](https://github.com/tabwrangler/tabwrangler/blob/3caa55d6e224142c0bcd3450aa5aea1aecd66154/webpack.config.js#L120-L143) configs in `webpack.config.js`. Can you move this into the Firefox-specific config so it is only added in FF? That way Chrome will not get duplicate shortcut keys.

**Review by ssorallen — CHANGES_REQUESTED** [review]
Please move the FF-specific config into the Webpack config so it only appears in FF.

File: app/manifest.template.json (hunk 1 above)
Comment (xiio) [reply app/manifest.template.json]: Thank you for pointing out! Done ✅

**Comment by ssorallen** [comment]
Thanks for the update, I tested in Firefox and it looks great.

**Pull request merged**